api/room/schema.py

```diff
@@ -183,7 +184,29 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class UpdateFirebaseToken(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+        firebase_token = graphene.String()
+    room = graphene.Field(Room)
+
+    def mutate(self, info, room_id, **kwargs):
+        MRM_URL = "https://www.google.com/"
+        validate_empty_fields(**kwargs)
+        query_room = Room.get_query(info)
+        active_rooms = query_room.filter(RoomModel.state == "active")
+        room = active_rooms.filter(RoomModel.id == room_id).first()
```

Comment:
Also, is the .first() required here seeing as the room_ids are unique?

Reply:
let me test it out

Reply:
> Also, is the .first() required here seeing as the room_ids are unique?

@RonKbS , Without the .first() it returns a Query and not an object. We actually need the object in this case.